### ISSUE TITLE
Game Concept and Overall Evaluation

### ISSUE THREAD

**Issue by KevinLane36** [Kmoran1512/Math-Battle#14]
Great job on the game concept. I think this has the makings for a really fun game.
I loved the art and the writeups. 

I think the game itself is not fully there yet. I am not sure what is happening each round and I don't know which minions I have or don't have. 
I am also not sure how to purchase minions or the results of the fight.
Are we supposed to be using a certain voice over software?

**Comment by Kmoran1512** [comment]
Currently most of the game outputs and updates are done in the console, but I understand that is quite convoluted. We are currently working to make the game more understandable.

Thank you for all of your help

**Comment by shayandolikhani** [comment]
I agree it is a great concept and the art is great, but there are issues with how understandable it is. The instructions do seem a little convoluted, and when I am playing the game I am unsure if I am doing it right or if anything is happening, perhaps adding some visual or audio feedback in response to user actions each round would help with this. Additionally, some of the images don't load for some reason.

**Comment by annaserenius4** [comment]
The concept of this game is super awesome and I love all of the artwork, however I was unsure how to actually get it to start! I didn't know if I should be clicking or using the keyboard. I tried dragging the piece of gold on top of the minion card so I could buy it, but that didn't do anything:(
